BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift

```diff
@@ -81,7 +81,10 @@ extension DigitalInvoiceCoordinator: DigitalInvoiceViewModelDelagate {
     }
 
     func didTapPay(on viewModel: DigitalInvoiceViewModel) {
-        delegate?.didFinishAnalysis(self, invoice: viewModel.invoice, analysisDelegate: analysisDelegate)
+        if let analysisDelegate = analysisDelegate {
```

Comment:
[clarification]: what happens if `analysisDelegate` is nil ? Do we need to handle the scenario?

Reply:
we didn't handle that case in the past - nothing will happened if delegate is nil